### DISCUSSION TITLE
Question about how much Energy storage would be needed to support 100% renewables in the NY ISO

### DISCUSSION THREAD

**Discussion by haribis1** [NREL/ReEDS-2.0#86] [Q&A: Execution]
Would like to model the Question how much Energy storage would be needed to support 100% renewables in the NY ISO

**Comment by vcelloho** [comment]
It depends on the exact policy you want to represent,  you can alter the NY rps_all field so that it goes to 100% instead of 76%.  This operates in addition to the NY 100% Clean Energy Standard which is managed in the ces_fracion.csv file.

https://github.com/NREL/ReEDS_OpenAccess/blob/master/inputs/state_policies/recperc.csv
/inputs/state_policies/recperc.csv

If you want to model a 100% instate renewable policy you will also need to adjust the out of state allowed REC fraction. Which is controlled by oosfrac.txt.

https://github.com/NREL/ReEDS_OpenAccess/blob/master/inputs/state_policies/oosfrac.txt

A 100% RPS implemented in NY will allow other states to rely upon other states for providing capacity for resource adequacy.

**Reply by haribis1** [reply]
@vcelloho  thanks for your input. I would actually prefer to run the 76% RPS  first and then compare with 100% .

**Reply by haribis1** [reply]
is there report I can reference for the NY 76% RPS ?

**Reply by wesleyjcole** [reply]
The RPS requirements in ReEDS are derived from the data published by LBNL at https://emp.lbl.gov/publications/us-renewables-portfolio-standards-3

**Reply by haribis1** [reply]
@wesleyjcole thanks for sharing.  I was looking for how much storage will be needed to support 76% RE , not in the report .

**Reply by vcelloho** [reply]
@haribis1 the 76% in ReEDS is including calculations that adjust the RPS for the different types of utilities covered by the NY RPS/CES policy. Runs using this assumption are included in the annual Standard Scenarios Report. Results from the runs is available here and you can check the storage capacities under different scenarios for NY.
https://cambium.nrel.gov/?project=c85d86ff-f6ec-4812-925b-ceb9a1465506&mode=download&layout=DAC%20View

**Comment by haribis1** [comment]
thanks .
________________________________
